paddle/fluid/framework/ir/pass_tester_helper.h

```diff
@@ -184,12 +184,11 @@ struct Layers {
     return unary_op("c_identity", x, out, &attrs);
   }
 
-  VarDesc* c_allreduce_sum(VarDesc* x,
-                           VarDesc* out = nullptr,
-                           int ring_id = -1) {
+  VarDesc* all_reduce(VarDesc* x, VarDesc* out = nullptr, int ring_id = -1) {
```

Comment:
all_reduce  ->  all_reduce_sum, and modify all calls in `fused_multi_transformer_decoder_pass_tester.cc`~

Reply:
Thx, the variable names has been modified.